packages/manager/apps/telecom/src/app/telecom/pack/slots/hostedEmail/detail/translations/Messages_en_GB.json

```diff
@@ -49,9 +49,8 @@
   "hosted_email_detail_client_delete_account_confirm": "Confirm deletion",
   "hosted_email_detail_client_delete_account_success": "Votre compte {{email}} est en cours de suppression.",
   "hosted_email_detail_client_delete_account_error": "Oups ! La suppression du compte a échoué.",
-  "hosted_email_detail_individual": "Hosted 25GB",
   "input_password_strength": "Password strength:",
   "input_password_danger": "Weak",
   "input_password_warning": "Average",
   "input_password_success": "High"
-}
+}
```

Comment:
I'm suggesting the following fix into our lint-staged configuration #2918
in order stop having `No newline at end of file` report.